person.rb

```diff
@@ -0,0 +1,23 @@
+class Person
+  attr_accessor :name, :age
+  attr_reader :id
+
+  def initilize(age, name = 'Unknown', parent_permission: true)
+    @id = rand(1_000_000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  def can_use_services?
+    return true if of_age? || @parent_permission
+
+    false
```

Comment:
**OPTIONAL** - To help make your code cleaner, the `of_age?` method is returning a `bool` value, so I suggest shortening your code to `of_age? || @parent_permission`. This would help make your code shorter and easier to read, please fix this 🙏

Reply:
Thank you